arviz/tests/external_tests/test_data_pyjags.py

```diff
@@ -16,6 +20,41 @@
 PYJAGS_PRIOR_DICT = {"b": np.random.randn(3, 10, 3), "int": np.random.randn(1, 10, 3)}
 
 
+EIGHT_SCHOOL_DATA = {
+    "J": 8,
+    "y": np.array([28.0, 8.0, -3.0, 7.0, -1.0, 1.0, 18.0, 12.0]),
+    "sigma": np.array([15.0, 10.0, 16.0, 11.0, 9.0, 11.0, 10.0, 18.0]),
+}
+
+EIGHT_SCHOOL_PRIOR_MODEL_CODE = """
+model {
+    mu ~ dnorm(0.0, 1.0/25)
+    tau ~ dt(0.0, 1.0/25, 1.0) T(0, )
+    for (j in 1:J) {
+        theta_tilde[j] ~ dnorm(0.0, 1.0)
+    }
+}
+"""
+
+EIGHT_SCHOOL_POSTERIOR_MODEL_CODE = """
```

Comment:
Same comment for this posterior code as made on the prior code

Reply:
Done